smdebug/profiler/system_profiler_file_parser.py

```diff
@@ -51,16 +52,34 @@ def read_events_from_file(self, eventfile):
         try:
             with open(eventfile) as json_data:
                 event_line = json_data.readline()
+                lineNum = 0
                 while event_line:
+                    lineNum += 1
                     json_event = json.loads(event_line)
                     event_line = json_data.readline()
                     self.read_event_from_dict(json_event)
         except Exception as e:
+            with open(eventfile) as j_data:
+                event_line = j_data.readline()
+                lineNumTotal = 0
+                while event_line:
+                    event_line = j_data.readline()
+                    lineNumTotal += 1
+            errStr = 'line No. ' + str(lineNum) + ' out of total ' + str(lineNumTotal) + ' lines, '
+            cur_dir = os.path.dirname(eventfile)
+            cur_dir_base = os.path.basename(cur_dir)
+            eventfile_base = os.path.basename(eventfile)
+            parent_dir = os.path.dirname(cur_dir)
+            time_stamp_files = [int(x.split('.')[0]) for x in os.listdir(cur_dir)]
+            time_stamp_dirs = [int(x) for x in os.listdir(parent_dir)]
+            file_pos = time_stamp_files.index(int(eventfile_base.split('.')[0]))
+            dir_pos = time_stamp_dirs.index(int(cur_dir_base))
+            errStr += 'file No.' + str(file_pos+1) + ' in file No.1 to No.' + str(len(time_stamp_files)) + ', folder No.' + str(dir_pos+1) + " in folder No.1 to No." + str(len(time_stamp_dirs)) + "\n"
```

Comment:
This is code that's publicly exposed to all customers. We should make the error message a lot more descriptive, and also, instead of doing `len(time_stamp_dirs)`,  just set the original variable as the length

Reply:
actually nvm, the len(time_stamp_dirs) is fine, but the string isn't very descriptive.
Also, nit but this line of code is too long. Can you break it up into separate lines?

Reply:
pushed in changes to make the code more descriptive. Also, breaking up the long line.